features/full_tests/batch_2/release_stage.feature

```diff
@@ -13,15 +12,6 @@ Scenario: Exception reported when release stages empty
     When I run "EmptyEnabledReleaseStageScenario"
     Then I should receive no requests
```

Comment:
Log assertions should be able to help with these scenarios, which was noted down as part of PLAT-5908. I'll be implementing the functionality as part of my sprint so happy to merge this PR as long as that ticket isn't closed yet.

Reply:
> Log assertions should be able to help with these scenarios, which was noted down as part of PLAT-5908. I'll be implementing the functionality as part of my sprint so happy to merge this PR as long as that ticket isn't closed yet.

Ah yes, looks like I missed the last sentence of the ticket.  I'll update it accordingly and wind its state back.